modules/dev-server/variables.tf

```diff
@@ -73,3 +73,9 @@ variable "assign_public_ip" {
   type        = bool
   default     = false
 }
+
+variable "tls" {
+  description = "Whether to enable TLS on the server."
```

Comment:
Could be expanded to explain it's for Consul control plane comms to differentiate from Connect.